src/plugins/notebook/components/Notebook.vue

```diff
@@ -456,12 +522,12 @@ export default {
             this.sectionsChanged({ sections });
         },
         newEntry(embed = null) {
-            this.search = '';
+            this.resetSearch();
```

Comment:
not sure why I added this 2 times 😅

Reply:
Haha, I wasn't either, BUT just left it and figured you'd let me know if I could remove it. Does it matter if I remove the first or the lest call?